ix-dev/community/versitygw/templates/docker-compose.yaml

```diff
@@ -0,0 +1,92 @@
+{% set tpl = ix_lib.base.render.Render(values) %}
+
+{% set c1 = tpl.add_container(values.consts.versitygw_container_name, "image") %}
+
+{% set perm_container = tpl.deps.perms(values.consts.perms_container_name) %}
+{% set perms_config = {"uid": values.run_as.user, "gid": values.run_as.group, "mode": "check"} %}
+
+{% do c1.set_user(values.run_as.user, values.run_as.group) %}
+{% do c1.healthcheck.set_test("wget", {
+  "port": values.network.api_port,
+  "path": "/healthz",
+}) %}
+
+{% if not values.network.host_network %}
+  {% do c1.ports.add_port(values.network.api_port, values.network.api_port) %}
+  {% if values.network.admin_port %}
+    {% do c1.ports.add_port(values.network.admin_port, values.network.admin_port) %}
+  {% endif %}
+{% endif %}
+
+{% do c1.add_storage(values.consts.buckets_path, values.storage.buckets) %}
+{% do perm_container.add_or_skip_action("buckets", values.storage.buckets, perms_config) %}
+
+{% for store in values.storage.additional_storage %}
+  {% do c1.add_storage(store.mount_path, store) %}
+  {% do perm_container.add_or_skip_action(store.mount_path, store, perms_config) %}
+{% endfor %}
+
+{% set envs = namespace(x={
+  "ROOT_ACCESS_KEY_ID": values.versitygw.root_user_access_key,
+  "ROOT_SECRET_ACCESS_KEY": values.versitygw.root_user_secret_access_key,
+}) %}
+
+{% for key, value in envs.x.items() %}
+  {% do c1.environment.add_env(key, value) %}
+{% endfor %}
+
+{# TODO: could add additional env vars here #}
```

Comment:
I didn't need this right now, so I left it off; I can remove the TODO if you want. Most of [the options](https://github.com/versity/versitygw/wiki/Global-Options) can be configured with either a flag or environment variable.